cli/cmd/cluster/cluster.go

```diff
@@ -358,6 +358,7 @@ func (c controlplaneUpdater) ensureComponent(component, namespace string) error
 
 	rollback.Wait = true
 	rollback.Version = history[0].Version
+	rollback.MaxHistory = 10
```

Comment:
10 is the default limit used by the `helm rollback` command. We can discuss if 10 isn't too much and want something like 5 releases instead.

```
--history-max int    limit the maximum number of revisions saved per release. Use 0 for no limit (default 10)
```